services/common/src/customPropTypes/common.js

```diff
@@ -0,0 +1,19 @@
+import { PropTypes, shape, arrayOf, oneOfType } from "prop-types";
```

Comment:
I'm not sure about bringing in customPropTypes to common when we're phasing it out. What can we do to remove it instead?

Reply:
This is removed.